src/Licenses/Entities/License/PredictionData.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace ArrowSphere\PublicApiClient\Licenses\Entities\License;
+
+use ArrowSphere\PublicApiClient\AbstractEntity;
+use ArrowSphere\PublicApiClient\Exception\EntityValidationException;
+
+/**
+ * Class ActiveSeats
+ */
+class PredictionData extends AbstractEntity
```

Comment:
As I suggested before
```suggestion
class Prediction extends AbstractEntity
```